en/docs/administer/logging-and-monitoring/logging/setting-up-logging.md

```diff
@@ -93,6 +93,36 @@ Please follow below steps to enable logs for a given service component available
     loggers = org-wso2-carbon-user-core, trace-messages, org-apache-coyote,com-hazelcast
     ```
 
+### Enable logs for a component via the UI
+
+!!! note
+You need to get the [latest product updates for your product]({{base_path}}/administer/updating-wso2-api-manager/) to use this feature in the current version of WSO2 API-M. This feature is available as a product update in WSO2 API-M 3.2.0 from March 1, 2022 onwards.
```

Comment:
Not sure how this note renders properly coz the content should start after 4 spaces

Reply:
Fix in 1118bdd36c1dc9b46958f12176e14252fef9e004